pkg/network/http/http_statkeeper.go

```diff
@@ -0,0 +1,73 @@
+// +build linux_bpf
+
+package http
+
+import (
+	"strings"
+	"sync"
+
+	"github.com/DataDog/datadog-agent/pkg/process/util"
+)
+
+type Key struct {
+	SourceIP util.Address
+	DestIP   util.Address
+	DestPort uint16
```

Comment:
As mentioned in our chat yesterday, if you disregard SourcePort you're in effect losing container resolution

Reply:
oops yes I did make that change but just forget to add it as part of this commit 😬   it's there now!